README.md

```diff
@@ -1 +1,78 @@
-# Webhook Module for PHP
\ No newline at end of file
+# Webhook Module for PHP
+
+Adyen library for handling notification webhooks.
+
+## Installation
+You can use [Composer](https://getcomposer.org/). Follow the [installation instructions](https://getcomposer.org/doc/00-intro.md) if you do not already have composer installed.
+
+~~~~ bash
+composer require adyen/php-webhook-module
+~~~~
+
+In your PHP script, make sure you include the autoloader:
+
+~~~~ php
+require __DIR__ . '/vendor/autoload.php';
+~~~~
+
+Alternatively, you can download the [release on GitHub](https://github.com/Adyen/php-webhook-module/releases).
+
+## Usage
+
+#### Authenticate and validate incoming webhook request:
+~~~~ php 
+// Setup NotificationReceiver with dependency injection or create an instance as follows
+$notificationReceiver = new \Adyen\Webhook\Receiver\NotificationReceiver(new \Adyen\Webhook\Receiver\HmacSignature);
+
+// Authorize notification
+if (!$notificationReceiver->isAuthenticated(
+    $request['notificationItems'][0]['NotificationRequestItem'],
+    YOUR_MERCHANT_ACCOUNT,
+    YOUR_NOTIFICATION_USERNAME,
+    YOUR_NOTIFICATION_PASSWORD
+)) {
+    throw new AuthenticationException();
+}
+
+// Process each notification item
+foreach ($request['notificationItems'] as $notificationItem) {
+    // validate the notification
+    if ($notificationReceiver->validateHmac($notificationItem, YOUR_HMAC_KEY)) {
+       // save notification to your database
+       $this->databaseService->saveNotification($notificationItem); 
+    }
+}
+
+return new JsonResponse('[accepted]');
+~~~~
+
+#### Process notification to get new payment state:
+~~~~ php 
+$notificationItem = \Adyen\Webhook\Notification::createItem([
+    'eventCode' => $notification['eventCode'],
+    'success' => $notification['success']
+]);
+
+$processor = \Adyen\Webhook\Processor\ProcessorFactory::create(
+    $notificationItem,
+    $currentPaymentState,
+    $this->logger
+);
+
+$newState = $processor->process();
+~~~~
+NB: set `$currentPaymentState` to one of the values in `\Adyen\Webhook\PaymentStates`
```

Comment:
What is NB here?

Reply:
nota bene

Reply:
ok bene

Reply:
I also didn't recognise it at first, does it make sense to change it to this below:

```suggestion
Nota Bene: set `$currentPaymentState` to one of the values in `\Adyen\Webhook\PaymentStates`
```